staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service.go

```diff
@@ -166,3 +161,23 @@ func (g *gRPCService) Encrypt(plain []byte) ([]byte, error) {
 	}
 	return response.Cipher, nil
 }
+
+func (g *gRPCService) interceptor(
+	ctx context.Context,
+	method string,
+	req interface{},
+	reply interface{},
+	cc *grpc.ClientConn,
+	invoker grpc.UnaryInvoker,
+	opts ...grpc.CallOption,
+) error {
+	if method != "/v1beta1.KeyManagementService/Version" {
+		if err := g.checkAPIVersion(ctx); err != nil {
+			return err
+		}
+	}
+
+	err := invoker(ctx, method, req, reply, cc, opts...)
+	klog.V(5).Infof("Invoked method:%s, error:%v", method, err)
```

Comment:
do we really want to log on every encrypt/decrypt call? this seems so verbose as to be useless